cluster/gke/util.sh

```diff
@@ -293,8 +293,10 @@ function detect-node-names {
 
 # Detect instance group name generated by gke.
 #
-# Note that this will only select instance groups in the same zone as the
-# cluster, meaning that it won't include all groups in a multi-zone cluster.
+# Note that the NODE_INSTANCE_GROUPS var will only have instance groups in the
+# same zone as the cluster, meaning that it won't include all groups in a
+# multi-zone cluster. The NODE_INSTANCE_GROUPS_URLS will contain all the
```

Comment:
ALL_INSTANCE_GROUP_URLS, perhaps? Otherwise anyone that doesn't read the comment will assume they're the same. Or perhaps they shouldn't even be exposed outside of the method if they aren't used?